### PR TITLE
feat: access level

### PR DESCRIPTION
**Pull request by ningkaiqiang**
Modify the access level of models and algorithms.

### CODE REVIEW

File: Sources/FSRS/Models/FSRSModels.swift [hunk 1]
```diff
@@ -36,16 +36,16 @@ public enum Rating: Int, Codable, Equatable, CaseIterable  {
...
-    var due: Date?              // Date of the last scheduling
-    var stability: Double?      // Memory stability during the review
-    var difficulty: Double?     // Difficulty of the card during the review
-    var elapsedDays: Double     // Number of days elapsed since the last review
-    var lastElapsedDays: Double // Number of days between the last two reviews
-    var scheduledDays: Double   // Number of days until the next review
-    var review: Date            // Date of the review
+public struct ReviewLog: Equatable, Codable, Hashable {
+    public var rating: Rating          // Rating of the review (Again, Hard, Good, Easy)
+    public var state: CardState?       // State of the review (New, Learning, Review, public Relearning)
```
Comment (ishiko732) [inline Sources/FSRS/Models/FSRSModels.swift:41]:
```suggestion
    public var state: CardState?       // State of the review (New, Learning, Review, Relearning)
```

**Review by ishiko732 — APPROVED** [review]
LGTM

File: Sources/FSRS/Models/FSRSModels.swift (hunk 1 above)
Comment (ningkaiqiang) [reply Sources/FSRS/Models/FSRSModels.swift:41]: 👌

**Review by L-M-Sherlock — APPROVED** [review]
LGTM

**Pull request merged**